Dockerfile.wolfi

```diff
@@ -0,0 +1,7 @@
+FROM docker.elastic.co/wolfi/chainguard-base@sha256:9f940409f96296ef56140bcc4665c204dd499af4c32c96cc00e792558097c3f1
```

Comment:
We'll eventually switch to just publishing the wolfi images, right?
When that happens I'd suggest to use an `ARG` for the base image:

```
ARG BASE_IMAGE=alpine@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b
FROM $BASE_IMAGE
```

This way we can have a single Dockerfile which locally builts on top of Alpine, but from CI we can pass in the wolfi image as base image.

Nothing to do for now, just something to maybe keep in mind for when we drop publishing the alpine based image.

Reply:
> We'll eventually switch to just publishing the wolfi images, right?

I think so, that's something we are still discussing internally